packages/block-library/src/template-part/block.json

```diff
@@ -28,5 +28,6 @@
 		},
 		"__experimentalLayout": true
 	},
+	"style": "wp-block-template-part",
```

Comment:
Why you added this?

Reply:
No longer needed since https://github.com/WordPress/gutenberg/pull/32275 got merged, so I removed this one :+1: